[wip] direct to unboxed func ptr

smessmer opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#32376 [wip] direct to unboxed func ptr**
* #30370 [wip] Use codegen'ed unboxing wrappers

Differential Revision: [D19456053](https://our.internmc.facebook.com/intern/diff/D19456053/)

smessmer committed:
[wip] direct to unboxed func ptr



Differential Revision: [D19456053](https://our.internmc.facebook.com/intern/diff/D19456053/)

[ghstack-poisoned]

kostmo commented:
## :pill: CircleCI build failures summary and remediations
As of commit 504ee156:


* **1/1** failures introduced in this PR


## Detailed failure analysis

One may explore the probable reasons each build failed interactively [on the Dr. CI website](https://dr.pytorch.org/commit-details.html?sha1=504ee15635cf9b8647f2887316befbddfd3e7f95).

### :detective: 1 new failure recognized by patterns
The following build failures do not appear to be due to upstream breakage:
#### [![See CircleCI build](https://avatars0.githubusercontent.com/ml/7?s=12)](https://circleci.com/gh/pytorch/pytorch/4294182) pytorch_linux_xenial_py3_6_gcc5_4_build (1/1)
**Step:** "Build" ([full log](https://dr.pytorch.org/api/view-log-full?build_id=87943246) | [pattern match details](https://dr.pytorch.org/build-details.html?build_id=87943246))
<details>
<summary>
<code>Automatic merge failed; fix conflicts and then commit the result.</code>
</summary>

```
HEAD is now at 504ee15635 [wip] direct to unboxed func ptr 
+ git reset --hard 504ee15635cf9b8647f2887316befbddfd3e7f95 
HEAD is now at 504ee15635 [wip] direct to unboxed func ptr 
+ git merge --allow-unrelated-histories --no-edit --no-ff bdd5e154370bff027d7f13e4cc685bd8d85f4de7 
Auto-merging aten/src/ATen/native/native_functions.yaml 
Auto-merging aten/src/ATen/function_wrapper.py 
Auto-merging aten/src/ATen/core/dispatch/Dispatcher.h 
Auto-merging aten/src/ATen/core/dispatch/Dispatcher.cpp 
CONFLICT (content): Merge conflict in aten/src/ATen/core/dispatch/Dispatcher.cpp 
Auto-merging aten/src/ATen/core/boxing/KernelFunction_impl.h 
Automatic merge failed; fix conflicts and then commit the result. 
```

</details>



---

<details><summary>This comment was automatically generated by <a href="https://github.com/kostmo/circleci-failure-tracker/tree/master/docs/from-pull-request-comment">Dr. CI</a> (expand for details).</summary>Follow <a href="https://dr.pytorch.org/admin/comments-opt-out.html">this link to opt-out</a> of these comments for your Pull Requests.

Please report bugs/suggestions on the <a href="https://github.com/kostmo/circleci-failure-tracker/issues">GitHub issue tracker</a>.
</details>

smessmer committed:
Update on "[wip] direct to unboxed func ptr"


Differential Revision: [D19456053](https://our.internmc.facebook.com/intern/diff/D19456053/)

[ghstack-poisoned]
